### PR TITLE
[issue 81] 

### PR DESCRIPTION
**Pull request by tyler-jachetta**
Initially creating PR with only failing test, will fill in with additional work as it is done
This new test fails, expressing the bug #81

### CODE REVIEW

**Comment by tyler-jachetta** [comment]
I think that these tests may also need to include versions with and without prefixes (v1.2.3 vs 1.2.3)

**Comment by otakup0pe** [comment]
I mean, the test fails. Will the fix be in this pr?

**Comment by tyler-jachetta** [comment]
I planned to do that once I did the work to fix the issue, but I can slice it up either way, can also just close this issue having referenced it in the issue (#81)

**Comment by otakup0pe** [comment]
Let's fix it in this PR so we aren't merging things with broken tests into mainline.

**Comment by tyler-jachetta** [comment]
oh, not what I meant, sorry. Two options I was presenting was use this PR, or use a fresh PR and drop this one (never committing/merging)

**Comment by otakup0pe** [comment]
Gotcha. I don't care either way, just don't want to see merging in things with broken tests heh.

**Comment by tyler-jachetta** [comment]
I can't for the life of me figure out why `mainline` doesn't exist in CI, @otakup0pe do you have any ideas/does `make test` off of this branch pass for you locally?

**Comment by tyler-jachetta** [comment]
Oh I think it might be because the branch isn't on `otakup0pe/avakas`, this makes sense

**Comment by tyler-jachetta** [comment]
and after all that Sturm und Drang, got the CI figured out. Looks like I _can_ merge this myself, but wouldn't mind a review as the changes are not minor @otakup0pe

**Comment by tyler-jachetta** [comment]
@otakup0pe do you want me to just trust myself here and merge? don't want to overly bother you, but also don't want to be too cavalier

File: avakas/avakas.py [hunk 1]
```diff
@@ -44,32 +44,41 @@ class Avakas():
...
+            if not version:
+                raise ValueError('Version must non-null/positive length')
+            if not isinstance(version, str):
+                raise TypeError("version must be type of str")
+            if self.tag_prefix and version.startswith(self.tag_prefix):
+                version = version[len(self.tag_prefix):]
+            try:
+                version = Version(version)
+            except ValueError as err:
+                # Doing this to get around the linter, which seems like a
```
Comment (otakup0pe) [inline avakas/avakas.py:74]: Which linter target specifically? Were you looking to skip a single line?

**Review by otakup0pe — APPROVED** [review]
Single question but given tests are passing I'm down with you merging it.

File: avakas/avakas.py (hunk 1 above)
Comment (tyler-jachetta) [reply avakas/avakas.py:74]: yeah, let me make the comment more clear, it's the line length, and it's pycodestyle which was having a problem iirc, I was having trouble getting the linter disabled on that single line for line length during the checks. If it's something I've missed that you know of, it was my first preference

Comment (tyler-jachetta) [reply avakas/avakas.py:74]: I'm going to merge, and create a new issue to tidy that up

**Pull request merged**